src/css/flags/media-audio.less

```diff
@@ -9,7 +9,7 @@
         }
     }
     // This has higher specificity to overwrite caption position when user inactive in playing state
-    &.jw-flag-user-inactive &.jw-state-playing {
+    &.jw-flag-user-inactive&.jw-state-playing {
```

Comment:
I think you can get rid of the ampersand too. Did you try?

``
&.jw-flag-user-inactive.jw-state-playing {
```